page-objects/pages/HMWRKAddEditHomeworkPage.js

```diff
@@ -1 +1,95 @@
-/*[url/homework/new] | [url/homework/[homeworkId]/edit]*/
\ No newline at end of file
+/*[url/homework/new] | [url/homework/[homeworkId]/edit]*/
+'use strict';
+const waitHelpers = require('../../runtime/helpers/waitHelpers.js');
+const dateTimeHelpers = require('../../runtime/helpers/dateTimeHelpers.js');
+const courseData = require('../../shared-objects/courseData');
+const courseHomeworksPage = require("../pages/coursePages/CRSSCourseHomeworksPage.js");
+
+const taskNameField = "input[placeholder='Titel']";
```

Comment:
let us unite selectors in const selectors = {selector1: 'xxx',}

Reply:
done